README.md

```diff
@@ -225,6 +320,7 @@ Make sure your IDE is listening for PHP debug connections and set up a path mapp
 2. Install the [PHP Debug](https://marketplace.visualstudio.com/items?itemName=felixfbecker.php-debug) extension.
 3. In your project, go to the Debug view, click "Add Configuration..." and choose PHP environment. A new launch configuration will be created for you.
 4. Set the `pathMappings` parameter to your local `wordpress` directory. Example:
+
 ```json
 "configurations": [
```

Comment:
@mitogh can we just delete this section since its duplicated above?

Reply:
Partially tho, as the main difference between versions 2 and 3 is the port. We can keep either and just mention the fact that the port should be updated appropriately instead.

Reply:
xdebug2 is no longer supported, its just xdebug3 now right? so I think we can remove it.

Reply:
actually you are correct, xdebug2 is still used for 7.1 and lower